lighthouse-core/gather/gatherers/initial-html.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const NetworkAnalyzer = require('../../lib/dependency-graph/simulator/network-analyzer.js');
+
+/**
+ * Collects the content of the initially-requested html document.
```

Comment:
```suggestion
 * Collects the content of the main html document.
```

"initial" makes me wonder if there were redirects then maybe this is the first actual document pre-redirect?   (afaik you can serve a response body with a 301, right?)  anyway....